<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>388</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QTableView" name="View">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="FilterBox">
     <property name="toolTip">
      <string>Select filter field.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="SearchField">
     <property name="placeholderText">
      <string>Search...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="DeleteButton">
       <property name="toolTip">
        <string>Click to delete selected row.</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/new/prefix1/list-remove.png</normaloff>:/new/prefix1/list-remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="AddButton">
       <property name="toolTip">
        <string>Click to add row.</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/new/prefix1/list-add.png</normaloff>:/new/prefix1/list-add.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>SearchField</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TabWidget</receiver>
   <slot>onSearchChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FilterBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>TabWidget</receiver>
   <slot>onSearchChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddButton()</slot>
  <slot>onDeleteButton()</slot>
  <slot>onSearchChanged()</slot>
 </slots>
</ui>
